addons/acr/config.cpp

```diff
@@ -0,0 +1,170 @@
+            "UKSF_ACR_BVP",
+            "UKSF_ACR_Buran_Transporter",
+            "UKSF_ACR_Buran_Medical",
+            "UKSF_ACR_Buran_Cargo",
+            "UKSF_ACR_Dingo_Mg",
+            "UKSF_ACR_Dingo_GL",
+            "UKSF_ACR_Tatra_Ammo",
+            "UKSF_ACR_Tatra_Transport",
+            "UKSF_ACR_Tatra_Fuel",
+            "UKSF_ACR_Tatra_MG",
+            "UKSF_ACR_Tatra_Repair",
+            "UKSF_ACR_RM_70",
+            "UKSF_ACR_Sholef",
+            "UKSF_ACR_Wiesel_AA",
+            "UKSF_ACR_Wiesel_Radar",
+
+            "UKSF_ACR_f35",
+            "UKSF_ACR_Gripen",
+            "UKSF_ACR_Mi35",
+            "UKSF_ACR_AH1Z",
+            "UKSF_ACR_Yabhon",
+            "UKSF_ACR_Li2",
+            "UKSF_ACR_Mi17",
+
+            "UKSF_ACR_Podnos",
+            "UKSF_ACR_ags",
+            "UKSF_ACR_M2_Raised",
+            "UKSF_ACR_M2_Low",
+            "UKSF_ACR_TOW"
+        };
+        weapons[]={
+            "UKSF_ACR_CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D",
+            "UKSF_ACR_CUP_arifle_ACRC_EGLM_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D",
+            "UKSF_ACR_LMG_03_F_CUP_acc_FlashlightACE_optic_MRCO_2D",
+            "UKSF_ACR_CUP_launch_RPG7V",
+            "UKSF_ACR_CUP_srifle_SVD_wdl_CUP_optic_PSO_1_1_open",
+            "UKSF_ACR_CUP_arifle_ACRC_blk_556",
+            "UKSF_ACR_CUP_arifle_ACRC_blk_556_ACE_optic_MRCO_2D",
+            "UKSF_ACR_CUP_hgun_CZ75",
+            "UKSF_ACR_CUP_srifle_M2010_blk_ACE_optic_LRPS_2D",
+            "UKSF_ACR_UKSF_ACR_ACR_CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D_CUP_acc_FlashlightACE_optic_MRCO_2D",
+            "UKSF_ACR_CUP_launch_FIM92Stinger_Loaded",
+            "UKSF_ACR_CUP_lmg_MG3_rail_ACE_optic_MRCO_2D",
+            "UKSF_ACR_launch_I_Titan_short_F",
+            "UKSF_ACR_"
+        };
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "uksf_main",
+            "uksf_common",
+
+            // not sure if these are needed??
+            "ace_medical_treatment",
+            "A3_Weapons_F",
+            "CUP_Weapons_ACR",
+            "CUP_Creatures_Military_ACR",
+            "A3_Characters_F",
+            "CuratorOnly_Characters_F_OPFOR",
+            "A3_Weapons_F_Exp_Machineguns_LMG_03",
+            "CUP_Weapons_RPG7",
+            "CUP_Weapons_SVD",
+            "A3_Characters_F_Tank_Headgear",
+            "CUP_Weapons_CZ75",
+            "CUP_Weapons_M2010",
+            "CUP_Weapons_Stinger",
+            "CUP_Weapons_MG3",
+            "A3_Weapons_F_Launchers_Titan"
+        };
+        author = QUOTE(UKSF);
+        authors[] = { "Bridgford.A" };
+        url = URL;
+        VERSION_CONFIG;
+    };
+};
+
+class CBA_Extended_EventHandlers;
+
+#include "CfgEventHandlers.hpp"
+#include "CfgGroups.hpp"
+#include "CfgVehicles.hpp"
+#include "CfgWeapons.hpp"
+#include "CfgFactionClasses.hpp"
+
+class CfgWorlds {
+    class GenericNames {
+        class CzechMen {
```

Comment:
Are these custom? If not, we should use whatever ones already exist and not redefine this

Reply:
They are custom names I added